src/Features/Core/Portable/GenerateType/AbstractGenerateTypeService.State.cs

```diff
@@ -247,7 +247,7 @@ private void SetBaseType(INamedTypeSymbol baseType)
 
                 // Strip off top-level nullability since we can't put top-level nullability into the base list. We will still include nested nullability
                 // if you're deriving some interface like IEnumerable<string?>.
-                BaseTypeOrInterfaceOpt = baseType.WithNullability(NullableAnnotation.None);
+                BaseTypeOrInterfaceOpt = (INamedTypeSymbol)baseType.WithNullableAnnotation(NullableAnnotation.None);
```

Comment:
❔ Why is this using `None` instead of `NotAnnotated`?

Reply:
@sharwell It doesn't really matter in this case.